cypress/integration/quickmark/instance-quickmark.spec.js

```diff
@@ -105,24 +101,23 @@ describe('Manage inventory Bib records with quickMarc editor', () => {
         InventoryInstance.deriveNewMarcBib();
         const expectedCreatedValue = QuickMarcEditor.addNewField();
 
-        // TODO: add return value to validate the result by value too. As minimum add catching from response. The best way - through interactors
-        QuickMarcEditor.deletePenaltField();
-
-        const expectedUpdatedValue = QuickMarcEditor.updateExistingField();
+        QuickMarcEditor.deletePenaltField().then(deletedTag => {
+          const expectedUpdatedValue = QuickMarcEditor.updateExistingField();
 
-        QuickMarcEditor.pressSaveAndClose();
-        QuickMarcEditor.deleteConfirmationPresented();
-        QuickMarcEditor.confirmDelete();
+          QuickMarcEditor.pressSaveAndClose();
+          QuickMarcEditor.deleteConfirmationPresented();
+          QuickMarcEditor.confirmDelete();
```

Comment:
```
QuickMarcEditor.pressSaveAndClose();
QuickMarcEditor.deleteConfirmationPresented();
QuickMarcEditor.confirmDelete();
```
Colud it be inside one function? Seems you need to reuse it several times

Reply:
additional confirmation dialog related with case when user removed feilds in Marc editor. These steps can be useful in tests level. I've added TODO related with these tests redesign.